README.rst

```diff
@@ -141,13 +159,119 @@ This authentication scheme uses Django's default session backend for authenticat
 Pagination
 ==========
 
-API pagination is provided with the help `page` parameter.
-The default page size is 10 which can be overridden using the `page_size` parameter.
+All *list* endpoints support the ``page_size`` parameter that allows paginating
+the results in conjunction with the ``page`` parameter.
+
+.. code-block:: text
+
+    GET /api/v1/firmware/build/?page_size=10
+    GET /api/v1/firmware/build/?page_size=10&page=2
+
+List of endpoints
+=================
+
+Since the detailed explanation is contained in the `Live documentation <#live-documentation>`_
+and in the `Browsable web page <#browsable-web-interface>`_ of each endpoint,
+here we'll provide just a list of the available endpoints,
+for further information please open the URL of the endpoint in your browser.
+
+Get next Available IP
```

Comment:
I don't understand, don't you see that these endpoints are already listed? https://github.com/codesankalp/openwisp-ipam/tree/api#get-next-available-ip-1 This new information is redundant, so is not needed, please remove it. If any information is missing from the current doc, let's add it to that, let's not duplicate it.

Reply:
I did this because in all other docs the list of endpoints is short and it is mentioned to use swagger for more details of the endpoints.
That's why I have added short lines.
But I revert it back.